blog-server-api/src/entities/post.rs

```diff
@@ -43,7 +27,17 @@ impl Into<Post> for ServicePost {
                 first_name: self.author_first_name,
                 last_name: self.author_last_name,
             },
-            tags,
+            tags: match self.tags {
+                Some(v) => v
+                    .iter()
```

Comment:
`.into_iter()` and then we can use struct pars as owned object

Reply:
done